src/dicom_constants.py

```diff
@@ -0,0 +1,7 @@
+"""Contains DICOM standard SOP class UID"""
+
+CT_IMAGE = "1.2.840.10008.5.1.4.1.1.2"
+RT_STRUCTURE_SET = "1.2.840.10008.5.1.4.1.1.481.3"
+RT_DOSE = "1.2.840.10008.5.1.4.1.1.481.2"
+RT_PLAN = "1.2.840.10008.5.1.4.1.1.481.5"
```

Comment:
RT_ION_PLAN is not included.
Early on in my participation, I added support for RT Ion Plan so that data I work with could be reviewed with OnkoDICOM.
Have we lost that?

Reply:
Slack message attached :)
https://t2022groupfire.slack.com/archives/C039CUK4AQ7/p1657877589923629